frontend/src/views/DiagnosticTunnel/index.vue

```diff
@@ -213,6 +216,15 @@ export default {
     disablePreviousButton() {
       return !this.previousStep || !this.formIsValid
     },
+    hasChanged() {
+      let hasChanged = false
+      Object.keys(this.payload).forEach((key) => {
```

Comment:
Yeah, getObjectDiff doesn't work in this case, because the payload only contains a subset of keys to begin with, without implying that those keys should be removed from the original object. But happy to add in a for loop with a break